src/XRP/issued-currency-client.ts

```diff
@@ -479,6 +479,32 @@ export default class IssuedCurrencyClient {
     )
   }
 
+  /**
+   * Unfreezes the trust line between this account (issuing account) and another account.
+   * Note that the trust line's limit is set to 0.
+   *
+   * @see https://xrpl.org/freezes.html#enabling-or-disabling-individual-freeze
+   *
+   * @param accountToUnfreeze The X-Address of the account which to unfreeze a trust line.
+   * @param currencyName The currency of the trust line to unfreeze.
+   * @param wallet The wallet unfreezing the trust line.
+   */
+  public async unfreezeTrustLine(
+    accountToUnfreeze: string,
+    currencyName: string,
+    wallet: Wallet,
+  ): Promise<TransactionResult> {
```

Comment:
Should there be an optional parameter of amount?

Reply:
I think that we can operate on the assumption that gateways will never be using trustlines with non-zero limits extended to a counterparty... so we can probably get await without an `amount` param.  However, if we find we need to change this in future we can just add the optional param as a non-breaking change so we can do less now.